### ISSUE TITLE
Can't escape out of mini file browser view

### ISSUE THREAD

**Issue by erohtar** [adirh3/Fluent-Search#419]
**Describe the bug**
Unlike other Alt functions from the search results, once you're in the mini file browser view, pressing escape does nothing. I have to use the mouse to go back to search field and press Esc, or just press Alt-F4 to exit. Even pressing Alt doesn't take me back!

And while we're on the subject of the mini file browser (amazing feature btw!), please provide options for more views like these:
![image](https://user-images.githubusercontent.com/5120628/149288504-62366218-ce85-48a0-be7a-f7590c1bb38d.png)

The default (and the only option right now) is huge icons and shows a very limited number of files at once.
Thank you (and sorry about the bombardment!)

**Comment by adirh3** [comment]
This issue has been potentially fixed in version 0.9.90.97

**Comment by adirh3** [comment]
This issue has been potentially fixed in version 0.9.90.97

**Comment by rajat-r** [comment]
The Details view is perfect!
And being able to Alt out of it was sorely needed, glad it works now!

Now, within other Alt views you can go to that pane using Right and come back using Left, and because in the details view you only traverse using up/down (not left/right), could you pls let Left key works so you come back to the result list in this case too? (You can keep it disabled for the icons view, for obvious reasons)

I also came across a bug that I don't recall happening earlier. 
I use the hotkey I've set to open a certain tag search (the one I set up per your guidance in Discord - I suggested you make a quick blog post of it), and I have FS open to just the search bar view, then when I just press down key, expecting it to expand and let me go through my preset results, it doesn't expand and the focus is lost from the search field. I'm guessing it's still going through the result list, but outside of view. 
It doesn't happen 100% of the times, but I'd say 3-4 out of 5.

**Comment by erohtar** [comment]
> I'd like to keep key usage consistent across all modes

I understand what you're saying, but right now both of them are inconsistent with the other Alt modes. This way, only one would be. And with my usage of it today, this is the mode that should be default and provides most value IMHO.

> Can you please open a different issue, getting a little confused with different issues at same page. As I need to ask for more information to understand what may cause it.

Yes, sorry, I should have! I'll create a topic, pls let me know if you need any more info than what I gave on Discord just now.

**Comment by adirh3** [comment]
> > I'd like to keep key usage consistent across all modes
> 
> I understand what you're saying, but right now both of them are inconsistent with the other Alt modes. This way, only one would be. And with my usage of it today, this is the mode that should be default and provides most value IMHO.
> 
> > Can you please open a different issue, getting a little confused with different issues at same page. As I need to ask for more information to understand what may cause it.
> 
> Yes, sorry, I should have! I'll create a topic, pls let me know if you need any more info than what I gave on Discord just now.

In which cases is it not consistent?

**Comment by erohtar** [comment]
I could've bet I've used Left key to come back to results from the Alt view that shows file properties, but I just tried it and it doesn't work - so I must've been mistaken.

Okay, would it be possible to at least make Alt key toggle work when in that Right pane? I mean the just pressing and releasing Alt key is not in use for any other purpose in that pane - so assigning it to also work in there can't hurt anything else.

I'm asking because countless times I've pressed Alt and nothing happened in there, I do it and realize I have to press Esc for this, but it's just natural to expect Alt to work there too when it works from search field and results list!
